R/diag_header.R

```diff
@@ -317,7 +288,7 @@ gcam14_colors<- c("Africa" = "navajowhite3",
                   "Former Soviet Union" = "plum2")
 
 
-rgb255 <- function(r, g, b) {rgb(r,g,b, maxColorValue=255)}
+rgb255 <- function(r, g, b) {grDevices::rgb(r,g,b, maxColorValue=255)}
```

Comment:
Apropos of our earlier discussion, it looks like the 32-region color palette is already in the package, so all we have to do is add it to the example in the vignette.